api/everest-server.gen.go

```diff
@@ -111,11 +111,18 @@ const (
 	UpgradeEngine UpgradeTaskPendingTask = "upgradeEngine"
 )
 
+// Defines values for CreateDatabaseClusterSecretParamsDbEngineType.
+const (
+	CreateDatabaseClusterSecretParamsDbEngineTypePostgresql CreateDatabaseClusterSecretParamsDbEngineType = "postgresql"
```

Comment:
can we create one common list of engine types (or import them from everest-operator) and avoid copy-pasting them each time we need to use them in API endpoint query params?

Reply:
This is auto-generated based on the swagger spec, not copy-pasted manually. If you look at the code implementation, we are using the consts from everest-operator API. I do not know how to tell the code-gen not to generate these each time ...